lib/trino-parquet/src/main/java/io/trino/parquet/reader/ColumnReaderFactory.java

```diff
@@ -25,6 +25,7 @@
 import io.trino.spi.type.AbstractVariableWidthType;
```

Comment:
no test?

Reply:
there are already tests covering it in TestFlatColumnReader and AbstractTestParquetReader